double_entry.gemspec

```diff
@@ -17,6 +17,31 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
+  gem.post_install_message = <<-'POSTINSTALLMESSAGE'
+Please note the following changes in DoubleEntry version 10.4:
```

Comment:
i wanted to use `DoubleEntry::VERSION` but if i interpolate this string, then the sample migration dies because it tries to interpolate `DoubleEntry.table_name_prefix` (which is weird, because it is defined...)